src/BizHawk.Emulation.Cores/Consoles/Nintendo/Gameboy/Gambatte.cs

```diff
@@ -101,7 +101,27 @@ public Gameboy(CoreComm comm, GameInfo game, byte[] file, Gameboy.GambatteSettin
 
 				if (_syncSettings.EnableBIOS)
 				{
-					bios = comm.CoreFileProvider.GetFirmware(biosSystemId, biosId, true, "BIOS Not Found, Cannot Load.  Change SyncSettings to run without BIOS.");
+					try
+					{
+						bios = comm.CoreFileProvider.GetFirmware(biosSystemId, biosId, true, "BIOS Not Found, Cannot Load.  Change SyncSettings to run without BIOS.");
+					}
+					catch (MissingFirmwareException)
+					{
+						if (_syncSettings.ConsoleMode != GambatteSyncSettings.ConsoleModeType.GBA)
+						{
+							throw;
+						}
+						else
+						{
+							//implicitly throws MissingFirmwareException if GBC BIOS is also missing
+							bios = comm.CoreFileProvider.GetFirmware(biosSystemId, "World", true, "BIOS Not Found, Cannot Load.  Change SyncSettings to run without BIOS.");

```

Comment:
Are we concerned that this leaves the "wrong" hash in the movie file?

Reply:
Yes, there's some discussion about moving this in-memory fallback into the firmware manager to reduce duplicate code between GBHawk and Gambatte